src/StoreKeeper/WooCommerce/B2C/Commands/SyncWoocommerceSingleProductImage.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace StoreKeeper\WooCommerce\B2C\Commands;
+
+use Exception;
+use StoreKeeper\WooCommerce\B2C\Exceptions\BaseException;
+use StoreKeeper\WooCommerce\B2C\Imports\ProductImageImport;
+use StoreKeeper\WooCommerce\B2C\Tools\TaskHandler;
+
+class SyncWoocommerceSingleProductImage extends SyncWoocommerceProductPage
+{
+    /**
+     * @return mixed|void
+     *
+     * @throws Exception
+     */
+    public function execute(array $arguments, array $assoc_arguments)
```

Comment:
can you add unit test for the use case 
notmal image -> cdn image -> normal image

Reply:
This file is removed. Added test on SyncWoocommerceSingleProduct instead.